src/Pretzel.Logic/Configuration.cs

```diff
@@ -38,7 +41,8 @@ internal Configuration()
             EnsureDefaults();
         }
 
-        internal Configuration(IFileSystem fileSystem, string sitePath)
+        [ImportingConstructor]
+        public Configuration(IFileSystem fileSystem, [Import("SourcePath")] string sitePath)
```

Comment:
The sitePath isn't used at all. Should I remove it as well?

Reply:
It is used to calculate the `_configFilePath`.

Reply:
That's correct. My bad. Before it wasn't imported, but now logic moved from `Program.cs` to `ReadFromFile`'s `OnImportsSatisfied`